docs/Reference/Collection/CollectionManipulation.md

```diff
@@ -348,3 +377,34 @@ const collection = client.collection('some-collection');
 await collection.drop();
 // the collection "some-collection" no longer exists
 ```
+
+## collection.enableCollecttionStream
+
+`async collection.enableCollecttionStream(hasStreamBody): Object`
+
+Updates the collection stream flag.
+
+**Arguments**
+
+- **data**: `Object`
+
+  An object with the following key-value:
+
+  - **hasStream**: `boolean` 
+
+    Whether the stream should be enabled on the collection or not.
+
+    This parameter must be set to `true` when enabling a stream on the collection.
+    You can't set the flag to `false` as on-demand stream deletion is not allowed.
+
+**Examples**
+
+```js
+const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
+//---- OR ----
+const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
+
+const collection = client.collection('some-collection');
+await collection.enableCollecttionStream( { hasStream: true } );
```

Comment:
you should directly take the boolean value and pass it to the API payload with a key defining there itself.

Reply:
Ok sure